nservicebus/azure-storage-queues/configuration.md

```diff
@@ -79,4 +79,11 @@ This feature can be enabled by specifying `.UseAccountNamesInsteadOfConnectionSt
 
 snippet:AzureStorageQueueUseAccountNamesInsteadOfConnectionStrings
 
-NOTE: This feature is not available in `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` Versions 6 and below.
\ No newline at end of file
+NOTE: This feature is not available in `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` Versions 6 and below.
+
+## Hashing algorithms
+If the queue is longer than [63 characters](https://msdn.microsoft.com/en-us/library/azure/dd179349.aspx), the Azure Storage Queues Transport uses a hashing algorithm to rename it. The default algorithm is `MD5`. In order to use `SHA1` instead, use the following configuration:
```

Comment:
@SimonCropp this was discussed when endpoint name became a mandatory thing. While core mandates the endpoint name, transport is the one that can check validity of the name. It feels as if this should be a dependency of the core on the transports to provide that validation.